mmrazor/structures/estimator/latency.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import time
+
+import torch
+
+
+def repeat_measure_inference_speed(model,
+                                   resource_args,
+                                   max_iter: int = 100,
+                                   log_interval: int = 100,
+                                   repeat_num: int = 1,
+                                   PRINT: bool = False) -> str:
+    """Repeat speed measure for multi-times to get more precise results."""
+    assert repeat_num >= 1
+
+    fps_list = []
+
+    for _ in range(repeat_num):
+
+        fps_list.append(
+            measure_inference_speed(model, resource_args, max_iter,
+                                    log_interval))
+
+    if repeat_num > 1:
+        fps_list_ = [round(fps, 1) for fps in fps_list]
+        times_pre_image_list_ = [round(1000 / fps, 1) for fps in fps_list]
+        mean_fps_ = sum(fps_list_) / len(fps_list_)
+        mean_times_pre_image_ = sum(times_pre_image_list_) / len(
+            times_pre_image_list_)
+        if PRINT:
+            print(
```

Comment:
use logger to print, with debug logger level.

Reply:
Done.